scripts/test

```diff
@@ -19,7 +19,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
         usage
     else
         docker \
-            run -d --name postgres-test -p 5432:5432 "quay.io/azavea/postgis:$POSTGIS_MAJOR-postgres$PG_MAJOR-$VARIANT"
+            run -d --name postgres-test -e POSTGRES_HOST_AUTH_METHOD=trust -p 5432:5432 "quay.io/azavea/postgis:$POSTGIS_MAJOR-postgres$PG_MAJOR-$VARIANT"
```

Comment:
This was necessary to get PostgreSQL 12.x builds to pass because we attempt a passwordless connection via `psql` in `test`.

Reply:
That makes sense given that we're trying to simulate a remote connection. Nice!